src/reader-activation-auth/index.js

```diff
@@ -63,13 +63,14 @@ window.newspackRAS.push( readerActivation => {
 				}
 			}
 			if ( redirect && redirect !== '#' ) {
-				callback = () => {
+				onSuccess = () => {
 					window.location.href = redirect;
 				};
 			}
 
 			openAuthModal( {
-				callback,
+				onSuccess,
+				onError: onSuccess,
```

Comment:
Let's rename the callback to something other than `onSuccess` since its meant to be called even when auth is unsuccessful. `callback` seemed fine to me.

Reply:
Updated in ba879387fa25db4eac9fd3eb016d7749952013e0